src/config/const.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    OPENAI_EMBEDDINGS_API: "https://api.openai.com/v1/embeddings",
+    OPENAI_CHAT_COMPLETIONS_API: "https://api.openai.com/v1/chat/completions",
+    OPENAI_EMBEDDINGS_MODEL: 'text-embedding-ada-002',
+    OPENAI_CHAT_COMPLETIONS_MODEL: "gpt-3.5-turbo",
+    OPENAI_CHAT_COMPLETIONS_MODEL_ROLE: "system",
+    OPENAI_CHAT_COMPLETIONS_CONTEXT: "You are a helpful assistant. Based on the context below, answer this specific query directly: {question} .You should always aim to be concise, helpful, and friendly in your responses. Do not ask follow-up questions or request more details.",
+    JIRA_URL: `https://paragprhr.atlassian.net/rest/api/2/issue`,
+    QUERY_SIMILARITY_ACCPETANCE_THRESHOLD: 0.8,
+    SLACK_PREAMBLE_TEMPLATE: `Hi {name},\nThank you for reaching out with your query. Here's what I found:\n\n`,
+    SLACK_POSTAMBLE_TEMPLATE: `\n\nFor further details, please visit the link: {url}.`,
+    JIRA_TICKET_TEMPLATE: `Hi {name},\n\n Thank you for bringing this to our attention. We've logged a Jira ticket to address your query. You can track the progress and updates on the following link: {ticketKey}.`,
+    SCRAPING_SECTION_URL_DIV_ID: '#categories-block-wrapper',
+    SCRAPING_ARTICLE_URL_DIV_ID: '#categories-block-wrapper',
+    SCRAPING_TITLE_DIV: 'h1',
+    SCRAPING_ARTICLES_DETAILS_DIV_ID: '#article-details',
+    SCRAPING_ARTICLE_CONTENT_DIV_ID: '#article-wrapper',
+    MAX_RETRIES: 3,
+    REQUEST_TIMEOUT: 5000,
+    CHAT_COMPLETIONS_MAX_TOKEN: 250,
+    CHAT_COMPLETIONS_TEMPERATURE: 0.3
+}
```

Comment:
consider static constants

Reply:
Right, this was a miss. Had a pending item of using ES modules instead of CommonJS and could have done export const <>: <>.
Will address this. Thanks

Reply:
keeping CommonJS for now. Made them static consts.